src/test/resources/features/ukvi/TeamRouting/MultipleContributions.feature

```diff
@@ -0,0 +1,174 @@
+@MultipleContributions @TeamRouting @UKVI @UKVIWorkflow
+Feature: Multiple Contributions
+
+  Background:
+    Given I log in to DECS
+    
+  Scenario: User can send a Triage case to the Triage (Contribution Requested) stage
+    And I create a "MPAM" case and move it to the "Triage" stage
+    And I load and claim the current case
+    And I send the Triage case to "Contributions Requested"
+    And I load and claim the current case
+    Then the case should be moved to the "Triage (Contribution Requested)" stage
+
+  Scenario: User can send a Draft case to the Draft (Contribution Requested) stage
+    And I create a "MPAM" case and move it to the "Draft" stage
+    And I load and claim the current case
+    And I send the Draft case to "Contributions Requested"
+    And I load and claim the current case
+    Then the case should be moved to the "Draft (Contribution Requested)" stage
+
+ Scenario Outline: User can mark a contribution as complete or cancelled at the Contributions Requested stage
+   And I create a "MPAM" case and move it to the "Triage" stage
+   And I load and claim the current case
+   And I send the Triage case to "Contributions Requested"
+   And I load and claim the current case
+   And I choose to "<action>" the contribution request at the multiple contribution stage
+   Then the contribution request should be displayed as "<action>"
+   Examples:
+     | action    |
+     | Complete  |
+     | Cancel    |
+
+ @UKVIRegression
+ Scenario Outline: User can select actions for contributions at the Triage - Contributions Requested stage
+   And I create a "MPAM" case and move it to the "Triage" stage
+   And I load and claim the current case
+   And I send the Triage case to "Contributions Requested"
+   And I load and claim the current case
+   And I choose to "<action>" the contribution request at the multiple contribution stage
+   And I select the "Contributions Received" action at the contributions requested stage
+   Then the case should be moved to the "Triage" stage
```

Comment:
Needs further assertions:

- Case is autoassigned to current user
- Workstack stage for case is 'Triage (Contributions received)'